File: Data Structure Usign C/Lab/Lab-1 14-07 Assignment-1/Task4.c
// Write a C program to define a structure 'employee' with members: id, name, and salary.
// Initialize the structure with values and display the id, name, and salary of the employee.

#include <stdio.h>
#include <conio.h>

struct employee
{
    int id;
    char name[20];
    float salary;
};

int main()
{
    struct employee e1 = {1011, "Sai", 2500000.00};
    printf("Employee Details\n");
    printf("Id: %d\n", e1.id);
    printf("Name: %s\n", e1.name);
    printf("Salary: %f\n", e1.salary);
    getch();
    return 0;
}

File: Advanced Web Technology/Lab/Lab-13 29-10 Assignment-13/Task4.php
<!-- 4.	Write a class called 'Employee' that extends the 'Person' class and adds properties like 'salary' and 'position'. Implement methods to display employee details -->

<?php
class Person {
    public $name;
    public $age;

    public function setName($name) {
        $this->name = $name;
    }

    public function setAge($age) {
        $this->age = $age;
    }

    public function getInfo() {
        return "Name: " . $this->name . ", Age: " . $this->age;
    }
}
class Employee extends Person {
    public $salary;
    public $position;

    public function set($position,$salary) {
        parent::setName("om mahadik");
        parent::setAge(30);
        $this->position = $position;
        $this->salary = $salary;
    }

    public function getInfo() {

       echo parent::getInfo() . ", Position: " . $this->position . ", Salary: $" . $this->salary;

    }
}

$employee = new Employee();
$employee->setPosition("Software Engineer");
$employee->setSalary(80000);
?>